tests/test_data/projects.json

```
@@ -0,0 +1,507 @@
+[
+    {
+        "_id": "XXX",
```

```suggestion
        "_id": "1234bc123125",
```
Change it for something unique in the JSON to be more informative